src/fondant/executor.py

```diff
@@ -39,13 +39,15 @@ def __init__(
         input_manifest_path: t.Union[str, Path],
         output_manifest_path: t.Union[str, Path],
         metadata: t.Dict[str, t.Any],
-        user_arguments: t.Dict[str, Argument],
+        user_arguments: t.Dict[str, t.Any],
+        output_partition_size: t.Optional[str] = "250MB",
```

Comment:
Shouldn't this be added to the parser as well, so it's extracted by `from_args`?